code/checks/language-codes.R

```diff
@@ -1,48 +1,26 @@
-# Check language codes
-# Author: Steven Moran <steven.moran@uzh.ch>
-# Date: April 2014
+# Check that the phoible ISO 639-3 codes are valid
 
-library(RMySQL)
+library(RCurl)
+library(dplyr)
 
-checkCodes <- function(df) {
-	codes <- df$LanguageCode
+# load data and standardize code column names
+iso639.3 <- read.csv("http://www-01.sil.org/iso639-3/iso-639-3.tab", sep="\t")
```

Comment:
If you're dealing with tab-delimited files, you can just use `read.delim()` (which defaults to tab-separation) instead of `read.csv(..., sep="\t")`.